src/getStats.js

```diff
@@ -0,0 +1,567 @@
+'use strict';
```

Comment:
Remove this file from code base, add it to demo dependencies only (use CDN for example) and describe in README.